### ISSUE TITLE
Please bump go from 1.17  to 1.18

### ISSUE THREAD

**Issue by jnguyen1195** [istio/istio#39844]
### Bug Description

We need to have new release patch as urgent with the go library bumped up to 1.18.1 from 1.17 since the CVEs which are listed below have  the critical  severities:

 

CVE-2022-29162

CVE-2022-28327

CVE-2022-24675

GHSA-qq97-vm5h-rrhg

CVE-2021-43784

Thank you

 

 

$ kubectl version --short

Flag --short has been deprecated, and will be removed in the future. The --short output will become the default.

Client Version: v1.24.0

Kustomize Version: v4.5.4

Server Version: v1.23.5

 

 

### Version

```prose
$ kubectl version --short

Flag --short has been deprecated, and will be removed in the future. The --short output will become the default.

Client Version: v1.24.0

Kustomize Version: v4.5.4

Server Version: v1.23.5
```

**Comment by ericvn** [comment]
Go 1.18 is used in Istio version 1.14, while Go 1.17 is used in older versions of 1.13 and 1.12 (the later goes End of Life in a few days). We typically do not move up Go major versions in point releases. We do however, typically pick up Go minor releases within our patch releases so Istio 1.13 and Istio 1.12 should have the latest Go 1.17 release available when they were built.

**Comment by howardjohn** [comment]
Half of the CVEs listed here are not golang CVEs. The rest are patched in 1.17.x, so no need to update, we already did: https://github.com/istio/tools/blob/b3e8b8ba95d4f5a997bb3fb152a07f1813e66a57/docker/build-tools/Dockerfile#L28. Thanks for the report. Closing since this is already resolved.
